cmd/edit.go

```diff
@@ -49,7 +49,7 @@ func RunEdit() *cobra.Command {
 				return err
 			}
 
-			if !!info.IsDir() {
+			if !info.IsDir() {
```

Comment:
This should be reverted or set to `if info.IsDr() {` since we want to check the content type and ignore dirs.

The `!!bool` syntax is useful sometimes, and confusing sometimes.